Cargo.toml

```diff
@@ -18,5 +18,7 @@ gumdrop_derive = "0.4"
 isatty = "0.1"
 lazy_static = "1"
 platforms = "0.1"
-rustsec = "0.9"
+rustsec = "0.9.3"
```

Comment:
@tarcieri I realized just moments after I changed this to 0.9.3 that maybe it should be less specific. What should it be?

…Or you could merge it and change it to be less specific in another commit…